content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst

```diff
@@ -0,0 +1,142 @@
+company’s recent purchases by total value, lead times, and number of RFQs sent.
+
+Filter and view options (e.g., Reference Number, Date, Vendor, etc.) allow toggling between
+different views to quickly sort and find orders. Just like in Odoo’s other apps, Filters and Groups
+can be customized in the search bar at the top of the page.
+
+Create new RFQ
+--------------
+
+To create a new RFQ, click the :guilabel:`New` button on the top left corner of the screen. A PO
+number will automatically be assigned to this new RFQ. From here, assign the vendor and order
+deadline, which lets the vendor know when they should confirm the quotation so that Odoo can
+confirm it into a PO.
+
+Additional fields
+~~~~~~~~~~~~~~~~~
+
+   :guilabel:`Vendor reference` points to the sales and delivery order numbers sent by the vendor.
+   This will come in handy once products are received and the PO needs to be matched to the delivery
+   order.
+
+   Under :guilabel:`Agreement`, blanket orders are long-term purchase agreements on recurring orders
+   with set pricing. To view and configure blanket orders, head to
+   :menuselection:`Purchase app --> Orders --> Purchase agreements`.
+
+   :guilabel:`Currency` can be changed if purchasing products from a vendor in another country.
+   To change the main currency or enable additional currencies, head to
+   :menuselection:`Configuration --> Settings --> Currencies`.
+
+   :guilabel:`Deliver to` provides options for the order shipment. Select the receiving warehouse
+   address here, or select :guilabel:`Dropship` to indicate that this order is to be shipped
+   directly to the end customer. When :guilabel:`Dropship` is selected, the
+   :guilabel:`Dropship address` field is enabled. Contact names will populate here from the
+   Contacts app.
+
+.. tip::
+   Expected Arrival is automatically calculated based on the Order Deadline and Vendor Lead Time.
+   Click the checkbox for :guilabel:`Ask confirmation` to ask for signage at delivery.
+
+In the products tab, add the products to be ordered. Select :guilabel:`Add a product` and start
+typing in the product name to select the item from the drop-down menu. :guilabel:`Catalog` can also
+be selected to navigate to a products menu under the chosen vendor. From here, products can be
+added to the cart. If ordering a product that has not been configured, type in the product name,
+select :guilabel:`Product Name` and manually add the unit price. Clicking into the product name
+navigates to its product page, where purchasing details for future orders can be configured.
+
+Send RFQ
+--------
+
+There are a few options to send an RFQ to vendors:
+
+:guilabel:`Send by Email` automatically loads an RFQ template email to send to the vendor’s email
+address (configured in Contacts). The subject and body can be directly edited from Odoo and the RFQ
+pdf can be edited before sending. Once the email has been sent, the order moves into the
```

Comment:
While it's definitely cool that we can send emails through Odoo, this doc is about the process of creating RFQs and |POs|. To keep the focus on that, I'm making a suggestion to gloss over these emailing features.

```suggestion
address (configured in the **Contacts** app). After crafting the desired message, click :guilabel:`Send`. Once the email has been sent, the |RFQ| moves to the
```